db/migrate/20240925160219_create_evidence_submissions.rb

```diff
@@ -6,12 +6,16 @@ def change
       t.string :request_id
       t.string :claim_id
       t.references :user_account, null: false, foreign_key: true, type: :uuid
+      # TODO: store first name, filename, doctype, and anything we want to show on the frontend or in emails
       t.json :template_metadata_ciphertext
       t.text :encrypted_kms_key
       t.string :upload_status
       t.string :va_notify_id
       t.string :va_notify_status
-      t.date :delete_date
+      t.datetime :delete_date
+      t.datetime :acknowledgement_date
+      t.datetime :failed_date
+      t.string :error_message
```

Comment:
Db migrations need to be in a separate PR.

Reply:
This is a PR to a feature branch. The DB changes will be in a separate PR when we merge to master